### PR TITLE
[ContactStructuralMechnicsApplication] Actual fix of the ULM condensed solver

### PR DESCRIPTION
**Pull request by loumalouomega**
**📝 Description**
Actual fix of the issue

**🆕 Changelog**
- This reverts commit 86d5964
- Fix indentation
- Update settings

### CODE REVIEW

File: applications/ContactStructuralMechanicsApplication/python_scripts/auxiliar_methods_solvers.py [hunk 1]
```diff
@@ -260,8 +260,45 @@ def  AuxiliarCreateLinearSolver(main_model_part, settings, contact_settings, lin
     if "ALMContactFrictional" in mortar_type or mortar_type == "ALMContactFrictionlessComponents":
         if contact_settings["use_mixed_ulm_solver"].GetBool():
             KM.Logger.PrintInfo("::[Contact Mechanical Solver]:: ", "Using MixedULMLinearSolver, definition of ALM parameters recommended")
-            mixed_ulm_solver = CSMA.MixedULMLinearSolver(linear_solver, contact_settings["mixed_ulm_solver_parameters"])
-            return mixed_ulm_solver
+            name_mixed_solver = contact_settings["mixed_ulm_solver_parameters"]["solver_type"].GetString()
+            if name_mixed_solver == "mixed_ulm_linear_solver":
+                print(name_mixed_solver)
```
Comment (AlejandroCornejo) [inline applications/ContactStructuralMechanicsApplication/python_scripts/auxiliar_methods_solvers.py]: leftover?

**Review by AlejandroCornejo — DISMISSED** [review]
It's the same conclusion? :)

File: applications/ContactStructuralMechanicsApplication/python_scripts/auxiliar_methods_solvers.py (hunk 1 above)
Comment (loumalouomega) [reply applications/ContactStructuralMechanicsApplication/python_scripts/auxiliar_methods_solvers.py]: Yep

Comment (loumalouomega) [inline applications/ContactStructuralMechanicsApplication/python_scripts/auxiliar_methods_solvers.py]:
```suggestion
```

Comment (loumalouomega) [reply applications/ContactStructuralMechanicsApplication/python_scripts/auxiliar_methods_solvers.py]: Need reapproval

**Review by AlejandroCornejo — APPROVED** [review]
imagen

**Comment by AlejandroCornejo** [comment]
you can merge @loumalouomega

**Pull request merged**